### PR TITLE
Update code generation commands in pipeline

### PR DESCRIPTION
**Pull request by houkms**
This pull request provides support for the new command line option "-c / --cloud" of Magic Modules. It updates the code generation commands in pipeline to specify the target cloud platform.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,11 +53,11 @@ build-terraform:
 	@if [ -z "$(RESOURCE)" ]; then \
 		echo "==> Generating source code for terraform from resource list..."; \
 		cd $(MMROOT) && \
-		jq '.[]' $(ROOT)/resources.json | xargs -I '{}' bundle exec compiler -d -p $(SPECROOT)/'{}' -e terraform -o $(TFROOT)/$(TFREPO)/; \
+		jq '.[]' $(ROOT)/resources.json | xargs -I '{}' bundle exec compiler -d -p $(SPECROOT)/'{}' -e terraform -c azure -o $(TFROOT)/$(TFREPO)/; \
```
Comment (metacpp) [inline Makefile]: Is `azure` switch newly introduced in `magic-modules` ? You've have to run `git submodule update --remote` to sync to latest commit, commit and push it.

File: Makefile [hunk 2]
```diff
@@ -71,11 +71,11 @@ build-ansible:
...
-		jq '.[]' $(ROOT)/resources.json | xargs -I '{}' bundle exec compiler -d -p $(SPECROOT)/'{}' -e ansible -o $(GENROOT)/$(ASREPO)/; \
+		jq '.[]' $(ROOT)/resources.json | xargs -I '{}' bundle exec compiler -d -p $(SPECROOT)/'{}' -e ansible -c azure -o $(GENROOT)/$(ASREPO)/; \
 	else \
 		echo "==> Generating source code for ansible from given name..."; \
 		cd $(MMROOT) && \
-		bundle exec compiler -d -p $(SPECROOT)/$(RESOURCE) -e ansible -o $(GENROOT)/$(ASREPO)/; \
+		bundle exec compiler -d -p $(SPECROOT)/$(RESOURCE) -e ansible -c azure -o $(GENROOT)/$(ASREPO)/; \
 	fi
 
 .PHONY: init clean build build-terraform format-terraform build-ansible
```
Comment (metacpp) [inline Makefile]: Can you add a newline ?

**Comment by metacpp** [comment]
@houkms thanks for the PR, please take a look at the failure in the Azure pipeline.

**Comment by JunyiYi** [comment]
Add related PR here: https://github.com/VSChina/magic-modules/pull/9/files.

File: Makefile (hunk 1 above)
Comment (houkms) [reply Makefile]: I noticed there is the _git submodule update_  already in [Makefile](https://github.com/Azure/magic-module-specs/blob/master/Makefile#L38). 

Is that means I need to update the submodule in my local repository?

**Comment by houkms** [comment]
This PR is closed due to github account permission issue. A new PR will be opened to realize the same function.

**Pull request closed without merging**